source/_integrations/go2rtc.markdown

```
@@ -0,0 +1,28 @@
+{% include integrations/config_flow.md %}
+
+If you are running Home Assistant with
+
+- {% term "Home Assistant Operating System" %}
+- {% term "Home Assistant Supervised" %}
+- {% term "Home Assistant Container" %}
+    
+than the go2rtc binary is already preinstalled in the container and the config flow will pick it automatically.
+
+If you are running {% term "Home Assistant Core" %}, than you need to setup a go2rtc instance and provide the URL in the config flow.
+Please refer to https://github.com/AlexxIT/go2rtc/
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)